### PR TITLE
A reader for Presidio fake PII aka benchmark data

### PR DESCRIPTION
**Pull request by gabechu**
### Description
A reader has been implemented to load benchmark data. Benchmark data is created by Presidio and it has a fix JSON format, the reader will parse that JSON and returns a `Data` object containing texts and labels.

#### Checklist
- [x] Added **tests** for changed code.
- [x] Mypy error of unbound TypeVar

### CODE REVIEW

File: pii_recognition/data_readers/data_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import pytest
+
+from .data import Data
+from pii_recognition.labels.schema import Entity
+
+
+def test_Data_token_labels():
```
Comment (torstenrudolf) [inline pii_recognition/data_readers/data_test.py:9]: these 2 tests seem to test `dataclass` functionality? I don't think they are useful for this project.

File: pii_recognition/data_readers/data_test.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        supported_entities={"PERSON", "BIRTHDAY"},
+        is_io_schema=False,
+    )
+    assert data.texts == texts
+    assert data.labels == labels
+    assert data.supported_entities == {"PERSON", "BIRTHDAY"}
+    assert data.is_io_schema is False
+
+
+def test_Data_for_texts_labels_length_mismatch():
```
Comment (torstenrudolf) [inline pii_recognition/data_readers/data_test.py:47]:
```suggestion
def test_data_for_texts_labels_length_mismatch():
```

File: pii_recognition/data_readers/presidio_fake_pii_reader.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+from typing import Dict, List, Set
+
+from pii_recognition.data_readers.data import Data
+from pii_recognition.utils import load_json_file
+
+
+class PresidioFakePiiReader:
+    def _get_supported_entities(self, data: List[Dict]) -> Set:
```
Comment (torstenrudolf) [inline pii_recognition/data_readers/presidio_fake_pii_reader.py:8]: are the entities of type `str`?
```suggestion
    def _get_supported_entities(self, data: List[Dict]) -> Set[str]:
```

File: pii_recognition/data_readers/data_test.py (hunk 1 above)
Comment (gabechu) [reply pii_recognition/data_readers/data_test.py:9]: That makes sense. Will remove the two tests.

File: pii_recognition/data_readers/presidio_fake_pii_reader.py (hunk 3 above)
Comment (gabechu) [reply pii_recognition/data_readers/presidio_fake_pii_reader.py:8]: You are right. At this point all three datasets we have are `str` label.

File: pii_recognition/data_readers/reader.py [hunk 4]
```diff
@@ -3,6 +3,7 @@
 from typing import List, Set
 
 
+# TODO: replace Data with the one from data.py
```
Comment (gabechu) [inline pii_recognition/data_readers/reader.py:6]: A separate card would be created for dealing with this.

File: pii_recognition/data_readers/data_test.py (hunk 2 above)
Comment (gabechu) [reply pii_recognition/data_readers/data_test.py:47]: Fixed! So it is a convention of using lower cases even tho we are testing a class?

**Comment by gabechu** [comment]
💇 @torstenrudolf Thank you for catching these!

File: pii_recognition/data_readers/data_test.py (hunk 2 above)
Comment (torstenrudolf) [reply pii_recognition/data_readers/data_test.py]: it's a python convention to use snake-case (all lower-case) for function names.

**Review by torstenrudolf — APPROVED** [review]

**Pull request merged**